dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/BaseTaskExecuteRunnable.java

```diff
@@ -52,4 +52,30 @@ public TaskExecutionContext getTaskExecutionContext() {
         return taskExecutionContext;
     }
 
+    @Override
+    public int compareTo(TaskExecuteRunnable other) {
+        if (other == null) {
+            return 1;
+        }
+        int workflowInstancePriorityCompareResult = workflowInstance.getProcessInstancePriority().getCode() -
+                other.getWorkflowInstance().getProcessInstancePriority().getCode();
+        if (workflowInstancePriorityCompareResult != 0) {
+            return workflowInstancePriorityCompareResult;
+        }
+
+        int taskInstancePriorityCompareResult = taskInstance.getTaskInstancePriority().getCode()
+                - other.getTaskInstance().getTaskInstancePriority().getCode();
+        if (taskInstancePriorityCompareResult != 0) {
+            return taskInstancePriorityCompareResult;
+        }
+        // larger number, higher priority
+        int taskGroupPriorityCompareResult =
+                taskInstance.getTaskGroupPriority() - other.getTaskInstance().getTaskGroupPriority();
+        if (taskGroupPriorityCompareResult != 0) {
+            return -taskGroupPriorityCompareResult;
```

Comment:
If larger number has higher priority, this place should return `taskGroupPriorityCompareResult `, and return value in line 63&69 should return opposite value.

Reply:
This is a history problem, the higher task group priority the small number.

Reply:
Ohh, so the comment should be `smaller number, higher priority`?

Reply:
Updated the comment, for task instance priority // smaller number, higher priority, for task group priority // larger number, higher priority

Reply:
Wait, two comments seem reversed, for task group priority, `smaller number, higher priority`, for workerflow/taskInstance, `larger number, higher priority`.

Reply:
No, for task instance/workflow instance smaller number, higher priority.

Reply:
I see, but according to UT and code logic, task instance of `Priority.HIGH` has higher priority than `Priority.LOW`

Reply:
Yes, Priority.HIGH have small number